tutorials/iot/ubuntu-kiosk/wayland-kiosk.md

```diff
@@ -505,33 +473,43 @@ Oh, still fails with a bunch of errors like
 
 
 ```
-libEGL warning: DRI2: failed to open swrast (search paths /usr/lib/x86_64-linux-gnu/dri:${ORIGIN}/dri:/usr/lib/dri)
-Error: eglInitialize() failed with error: 0x3001
+Error: eglGetDisplay() failed with error: 0x3000
+Error: eglGetDisplay() failed with error: 0x3000
 Error: main: Could not initialize canvas
 ```
 
 
-Courage! This is progress, we're almost done in fact. There's just one more thing to fix…
+Courage! This is progress, we’re almost done in fact. There’s just one more thing to fix…
+
 
 
 ## Common Problem 3: GL drivers are not where they usually are
 
 duration: 2:00
 
-This is another typical problem for snapping graphics applications: the GL drivers it needs are bundled inside the snap, but the application needs to be told the path to those drivers inside the snap.
+This is another typical problem for snapping graphics applications: the GL drivers it needs are bundled inside the snap, and the application needs to be configured to use these drivers.
 
-Is this "files are not where they're expected to be" yet again? Yes, but here we are lucky, there's an environment variable `$LIBGL_DRIVERS_PATH` we can use to point libGL to the correct location for the GL driver files it needs (you could use layouts too, but this is more efficient)
+Is this "files are not where they're expected to be" yet again? Yes, but here we are lucky as there’s environment variables we can use to specify the correct location for the GL driver files (you could use layouts too, but this is more efficient).
 
-So set `LIBGL_DRIVERS_PATH=$SNAP/usr/lib/x86_64-linux-gnu/dri` - and finally run
+Simply use:
 
 
 ```bash
-LIBGL_DRIVERS_PATH=$SNAP/usr/lib/x86_64-linux-gnu/dri \
-  $SNAP/command-glmark2-example.wrapper
+export __EGL_VENDOR_LIBRARY_DIRS="$SNAP/etc/glvnd/egl_vendor.d:$SNAP/usr/share/glvnd/egl_vendor.d"
```

Comment:
maybe add `root@in-snap:~#` for consistency with earlier?

Reply:
Good catch, done